### PR TITLE
separator option for .js concat

### PR DESCRIPTION
**Pull request by epferrari**
Hi, just made a small mod to respect the separator option passed to your task. Tested and working. Hope you merge, it'll save me from fixing sloppy history.js termination by hand on every freaking project ;)

### CODE REVIEW

File: tasks/bower-concat.js [hunk 1]
```diff
@@ -35,10 +35,11 @@ module.exports = function(grunt) {
 		var callback = this.data.callback;
 		var bowerOptions = this.data.bowerOptions || {};
 		var bowerDir = bowerOptions.relative !== false ? bower.config.cwd : '';
+		var options = this.options();
 
 		var done = this.async();
 		bowerMainFiles(function(jsFiles, cssFiles) {
-			concatenateAndWriteFile(jsFiles, jsDest);
+			concatenateAndWriteFile(jsFiles, jsDest, true);
```
Comment (sapegin) [inline tasks/bower-concat.js]: `concatenateAndWriteFile(jsFiles, jsDest, options.separator);`

No need for an extra verbose option.

File: tasks/bower-concat.js [hunk 2]
```diff
@@ -35,10 +35,11 @@ module.exports = function(grunt) {
 		var callback = this.data.callback;
 		var bowerOptions = this.data.bowerOptions || {};
 		var bowerDir = bowerOptions.relative !== false ? bower.config.cwd : '';
+		var options = this.options();
```
Comment (sapegin) [inline tasks/bower-concat.js]: It should have default value.

**Comment by sapegin** [comment]
Thanks! Could you also update the docs?

**Comment by epferrari** [comment]
Ok, I cleaned up the verbosity and declared default option value, updated the Readme, ran the linter, and added a test case. Should be good to go. Thanks!

**Comment by sapegin** [comment]
Thanks!

**Comment by epferrari** [comment]
You bet. Thanks for writing the module :)

**Pull request merged**